alacritty.yml

```diff
@@ -75,6 +75,9 @@ font:
   # Point size of the font
   size: 11.0
 
+  # Point size of the fallback font
+  fallback_size: 11.0
```

Comment:
Is this really worth another option? Wouldn't `size` be good enough?

Reply:
I find it very useful, because the unicode characters in the fallback font are sized differently from my main font. With this, I can configure them independently such that they are visually consistent.

Reply:
This is also something that I do as well. Although, it feels a little "extra", from a usability perspective it's very useful for getting things like icons in status bars (tmux, vim, etc) to look nice and aligned.